domainservice/src/test/kotlin/com/makarytskyi/rentcar/kafka/TestConfiguration.kt

```diff
@@ -0,0 +1,50 @@
+package com.makarytskyi.rentcar.kafka
+
+import com.makarytskyi.internalapi.subject.KafkaTopic
+import java.util.UUID
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.common.serialization.ByteArrayDeserializer
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.springframework.boot.autoconfigure.kafka.KafkaProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Scope
+import reactor.kafka.receiver.KafkaReceiver
+import reactor.kafka.receiver.ReceiverOptions
+
+@Configuration
+class TestConfiguration(
+    private val kafkaProperties: KafkaProperties,
+) {
+    @Bean("notificationReceiver")
+    @Scope("prototype")
```

Comment:
why prototype?

Reply:
To be able to run Kafka test in parallel.